sdk/search/azure-search-documents/README.md

```diff
@@ -38,7 +38,7 @@ Use the Azure Cognitive Search client library to:
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-search-documents</artifactId>
-    <version>11.4.0</version>
```

Comment:
Would we not need to update the POM file?

Reply:
Yep, forgot it